### PR TITLE
AB : AtragMx wiki updated

### PR DESCRIPTION
**Pull request by J3aLeiNe**
**AtragMx wiki updated**
- Add official manual and videos
- Add a quick example with the Atrahmx
- Add a quick example with the Truing tool

### CODE REVIEW

**Comment by J3aLeiNe** [comment]
Need a big, very big review.

File: docs/wiki/feature/atragmx.md [hunk 1]
```diff
@@ -14,19 +14,41 @@ version:
...
+https://www.youtube.com/watch?v=7SkRnbwoPmw
 
+## Usage
 
+Quick example with the M14 and the default 7.62mm 20Rnd Mag :
+
+ - Open the Range Card and check the cartridge, the zeroed distance, the rifle twist, 
+the Muzzle Velocity at 15°C and the Bore Height at the end.
+
+![Range Card](http://arma3.fr/files/media/user/204_45d986716bca9973.jpg)
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: If we could rehost that in our repo that would be great.

File: docs/wiki/feature/atragmx.md [hunk 2]
```diff
@@ -14,19 +14,41 @@ version:
...
 
-## Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings,
-instructions. May not apply to all modules.
+https://www.youtube.com/watch?v=7SkRnbwoPmw
 
+## Usage
 
+Quick example with the M14 and the default 7.62mm 20Rnd Mag :
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: No space before `:`.

File: docs/wiki/feature/atragmx.md [hunk 3]
```diff
@@ -14,19 +14,41 @@ version:
...
+ - Select `M` (Metric unit) at the top right.
+ - Open the `Gun` column, check and update the `Muzzle Velocity`, the `Zero Range` and `Done`.
+ 
+ 
+ **!!! The Muzzle Velocity Table will be automatically updated !!!**
+ - Optionally, `Save Gun` in the `GunList`.
+ - Actualise the `Atmsphr` column with the Kestrel and `Done` (Check the new Muzzle Velocity in the `Gun` column).
+ - Actualise the `Target` column.
+ - Apply the vertical and horizontal elevations on the scope.
+ - Control the breath and press ...
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: `...` in code tags?

File: docs/wiki/feature/atragmx.md [hunk 4]
```diff
@@ -14,19 +14,41 @@ version:
...
+ - `Open Gun` the 7.62x51mm M80 in the `GunList`.
+ - Select `E` (English unit) at the top right.
+ - Open the `Gun` column, check and update the `Bore`, the `Rifle Twist` and `Done`.
+ - Select `M` (Metric unit) at the top right.
+ - Open the `Gun` column, check and update the `Muzzle Velocity`, the `Zero Range` and `Done`.
+ 
+ 
+ **!!! The Muzzle Velocity Table will be automatically updated !!!**
+ - Optionally, `Save Gun` in the `GunList`.
+ - Actualise the `Atmsphr` column with the Kestrel and `Done` (Check the new Muzzle Velocity in the `Gun` column).
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: `Select` rather than `Actualise` maybe.

File: docs/wiki/feature/atragmx.md [hunk 5]
```diff
@@ -14,19 +14,41 @@ version:
...
+Quick example with the M14 and the default 7.62mm 20Rnd Mag :
+
+ - Open the Range Card and check the cartridge, the zeroed distance, the rifle twist, 
+the Muzzle Velocity at 15°C and the Bore Height at the end.
+
+![Range Card](http://arma3.fr/files/media/user/204_45d986716bca9973.jpg)
+
+ - Open the AtragMx and the `Atmsphr` column, select `Default` and `Done`.
+ - `Open Gun` the 7.62x51mm M80 in the `GunList`.
+ - Select `E` (English unit) at the top right.
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: `Imperial unit`

File: docs/wiki/feature/atragmx.md [hunk 6]
```diff
@@ -14,19 +14,41 @@ version:
...
-Short description of sub-feature 1.
+### AtragMx from HORUS VISION
+Horus ATragMX software considers atmospheric conditions, gun data, ammunition, range, speed and muzzle velocity to calculate precise aiming solutions with "come-up" results – and even accounts for Coriolis and spin drift effects. ATragMX, loaded on a handheld computer made by TDS Recon, is easy to use and lightning-fast. The Recon meets the rigorous MIL-STD-810F military standard for drops, vibration, humidity, altitude and extreme temperatures.
 
-### Sub-feature 2
-Short description of sub-feature 2.
+### Official Manuel and Horus Videos
+https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf
 
+https://www.youtube.com/watch?v=pg6oqT5jVds
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Use a link `[Title](link)`.

I'd put all videos at the end as references.

File: docs/wiki/feature/atragmx.md [hunk 7]
```diff
@@ -14,19 +14,41 @@ version:
...
 
-## Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings,
-instructions. May not apply to all modules.
+https://www.youtube.com/watch?v=7SkRnbwoPmw
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Use a link `[Title](link)`.

File: docs/wiki/feature/atragmx.md [hunk 8]
```diff
@@ -14,19 +14,41 @@ version:
...
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### AtragMx from HORUS VISION
+Horus ATragMX software considers atmospheric conditions, gun data, ammunition, range, speed and muzzle velocity to calculate precise aiming solutions with "come-up" results – and even accounts for Coriolis and spin drift effects. ATragMX, loaded on a handheld computer made by TDS Recon, is easy to use and lightning-fast. The Recon meets the rigorous MIL-STD-810F military standard for drops, vibration, humidity, altitude and extreme temperatures.
 
-### Sub-feature 2
-Short description of sub-feature 2.
+### Official Manuel and Horus Videos
+https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Use a link `[Title](link)`.

Also one empty line below header.

File: docs/wiki/feature/atragmx.md [hunk 9]
```diff
@@ -14,19 +14,41 @@ version:
 
 ## Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### AtragMx from HORUS VISION
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Empty line below header.

File: docs/wiki/feature/atragmx.md (hunk 5 above)
Comment (J3aLeiNe) [inline docs/wiki/feature/atragmx.md]: English units in the official manual, page 10.

File: docs/wiki/feature/atragmx.md (hunk 4 above)
Comment (J3aLeiNe) [inline docs/wiki/feature/atragmx.md]: The updating of both column `Atmsphr` and `Target` must be made in position.
Simply a "update" could be better ?

File: docs/wiki/feature/atragmx.md (hunk 6 above)
Comment (J3aLeiNe) [inline docs/wiki/feature/atragmx.md]: All the official documentation in the same place seems to me the best solution for a novice user.

File: docs/wiki/feature/atragmx.md (hunk 1 above)
Comment (J3aLeiNe) [inline docs/wiki/feature/atragmx.md]: Sorry but i don't know how to do that.

**Comment by J3aLeiNe** [comment]
Add a example with the truing tool and fix (hopefully) the PR.
But for the screenshot, i don't know how to do.

File: docs/wiki/feature/atragmx.md [hunk 10]
```diff
@@ -14,19 +14,63 @@ version:
...
 
-### Sub-feature 2
-Short description of sub-feature 2.
+Horus ATragMX software considers atmospheric conditions, gun data, ammunition, range, speed and muzzle velocity to calculate precise aiming solutions with "come-up" results – and even accounts for Coriolis and spin drift effects. ATragMX, loaded on a handheld computer made by TDS Recon, is easy to use and lightning-fast. The Recon meets the rigorous MIL-STD-810F military standard for drops, vibration, humidity, altitude and extreme temperatures.
 
+### Official Manual and Horus Videos
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Move this section to the end, just above Dependencies. And change to `### 3. <text>`

File: docs/wiki/feature/atragmx.md [hunk 11]
```diff
@@ -14,19 +14,63 @@ version:
...
+
+[Horus video part2](https://www.youtube.com/watch?v=7SkRnbwoPmw)
 
 ## Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings,
-instructions. May not apply to all modules.
+**Quick example with the M14 and the default 7.62mm 20Rnd Mag**
+
+*- Start of the mission:*
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `**Start of the mission:**`.

File: docs/wiki/feature/atragmx.md [hunk 12]
```diff
@@ -14,19 +14,63 @@ version:
...
+ - `Open Gun` the 7.62x51mm M80 in the `GunList`.
+ - Select `E` (English unit) at the top right.
+ - Open the `Gun` column, check and update the `Bore`, the `Rifle Twist` and `Done`.
+ - Select `M` (Metric unit) at the top right.
+ - Open the `Gun` column, check and update the `Muzzle Velocity`, the `Zero Range` and `Done`. 
+ 
+ **The Muzzle Velocity Table will be automatically updated**
+ - Optionally, `Save Gun` in the `GunList`.
+ 
+*- In position:*
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `**In position:**`.

File: docs/wiki/feature/atragmx.md [hunk 13]
```diff
@@ -14,19 +14,63 @@ version:
...
+ - Update the `Atmsphr` column with the Kestrel and `Done` (Check the new Muzzle Velocity in the `Gun` column).
+ - Update the `Target` column.
+ - Apply the vertical and horizontal elevations on the scope.
+ - Control the breath and press.
+ 
+**Quick example with the Truing tool**
+ 
+*- Start of the mission:*
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `**Start of the mission:**`.

File: docs/wiki/feature/atragmx.md [hunk 14]
```diff
@@ -14,19 +14,63 @@ version:
...
+ - Select `Drag Coef Table` in the `Options` menu.
+ - Add the `ZR` and the `C1` (`Gun` column) in the table and `Done`.
+  
+*- In Position:*
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `**In position:**`.

File: docs/wiki/feature/atragmx.md [hunk 15]
```diff
@@ -14,19 +14,63 @@ version:
...
+![Range Card](http://arma3.fr/files/media/user/204_45d986716bca9973.jpg)
 
+ - Open the AtragMx and the `Atmsphr` column, select `Default` and `Done`.
+ - `Open Gun` the 7.62x51mm M80 in the `GunList`.
+ - Select `E` (English unit) at the top right.
+ - Open the `Gun` column, check and update the `Bore`, the `Rifle Twist` and `Done`.
+ - Select `M` (Metric unit) at the top right.
+ - Open the `Gun` column, check and update the `Muzzle Velocity`, the `Zero Range` and `Done`. 
+ 
+ **The Muzzle Velocity Table will be automatically updated**
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `- **<text>**` without an empty line above, so it becomes inline as an italic indicator.

File: docs/wiki/feature/atragmx.md [hunk 16]
```diff
@@ -14,19 +14,63 @@ version:
...
+
+[Horus video part1](https://www.youtube.com/watch?v=pg6oqT5jVds)
+
+[Horus video part2](https://www.youtube.com/watch?v=7SkRnbwoPmw)
 
 ## Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings,
-instructions. May not apply to all modules.
+**Quick example with the M14 and the default 7.62mm 20Rnd Mag**
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `### 2.1 Example with M14 and 7.62mm 20Rnd Mag`

File: docs/wiki/feature/atragmx.md [hunk 17]
```diff
@@ -14,19 +14,63 @@ version:
...
 
+### Official Manual and Horus Videos
+
+[Official Manual](https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf)
+
+[Horus video part1](https://www.youtube.com/watch?v=pg6oqT5jVds)
+
+[Horus video part2](https://www.youtube.com/watch?v=7SkRnbwoPmw)
 
 ## Usage
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `## 2. usage`

File: docs/wiki/feature/atragmx.md [hunk 18]
```diff
@@ -14,19 +14,63 @@ version:
 
 ## Overview
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `## 1. Overview`

File: docs/wiki/feature/atragmx.md [hunk 19]
```diff
@@ -14,19 +14,63 @@ version:
...
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### AtragMx from HORUS VISION
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `### 1.1 ATragMX from Horus Vision`

File: docs/wiki/feature/atragmx.md [hunk 20]
```diff
@@ -14,19 +14,63 @@ version:
...
+ **The Muzzle Velocity Table will be automatically updated**
+ - Optionally, `Save Gun` in the `GunList`.
+ 
+*- In position:*
+ - Update the `Atmsphr` column with the Kestrel and `Done` (Check the new Muzzle Velocity in the `Gun` column).
+ - Update the `Target` column.
+ - Apply the vertical and horizontal elevations on the scope.
+ - Control the breath and press.
+ 
+**Quick example with the Truing tool**
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `### 2.2 Example with Truing tool`

File: docs/wiki/feature/atragmx.md [hunk 21]
```diff
@@ -14,19 +14,63 @@ version:
...
+ - Add the `ZR` and the `C1` (`Gun` column) in the table and `Done`.
+  
+*- In Position:*
+ - Open the `Truing Drop` in the `Options` menu.
+ - Add the actual `Target Range` in the `SUPER` column and `Calc`.
+ - Add the same `Target Range` in the `SUB` column  and `Calc`.
+ - Apply the actual scope elevation in the `Drop` field and `Calc`.
+ - `Accept` the new `C1`, `Gun` column and `Elev` are updated.
+ 
+ **The Drag Coefficient Table will be automatically updated**
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `- **<text>**` without an empty line above, so it becomes inline as an italic indicator.

File: docs/wiki/feature/atragmx.md [hunk 22]
```diff
@@ -14,19 +14,63 @@ version:
...
+ - Optionally, `Save Gun` in the `GunList`.
+ - If a new `Target Range` is applied in the `Target` column, select `Drag Coef Table` in the `Options` menu and `Done`.
+ - The ballistic coefficient `C1` and the elevation `Elev` will be recalculated.
+ 
+ 
 ## Dependencies
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Change to `## 4. Dependencies`.

**Comment by jonpas** [comment]
Added more comments to make it fit with the rest of the pages when it comes to style.

> But for the screenshot, i don't know how to do.

Put the image into `ACE3/docs/img/wiki/feature/` and link to it using:
```
<img src="{{ site.baseurl }}/img/wiki/feature/<image>" width="<w>" height="<h>" alt="<name>" />
```
replacing everything within `<>`.

File: docs/wiki/feature/atragmx.md [hunk 23]
```diff
@@ -12,21 +12,68 @@ version:
...
-## Usage
+### 2.1 Example with M14 and default 7.62mm 20Rnd Mag
 
-Short overview of how to use the feature, e.g. menu options, key bindings,
-instructions. May not apply to all modules.
+**Start of the mission:**
+ - Open the Range Card and check the cartridge, the zeroed distance, the rifle twist, 
+the Muzzle Velocity at 15°C and the Bore Height at the end.
 
+ img src="{{ site.baseurl }}/img/wiki/feature/atragmx1.jpg" width="431" height="600" alt="RangeCard" /
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Missing `<` at the beginning and `>` at the end.

File: docs/wiki/feature/atragmx.md [hunk 24]
```diff
@@ -12,21 +12,68 @@ version:
...
+**In position:**
+ - Open the `Truing Drop` in the `Options` menu.
+ - Add the actual `Target Range` in the `SUPER` column and `Calc`.
+ - Add the same `Target Range` in the `SUB` column  and `Calc`.
+ - Apply the actual scope elevation in the `Drop` field and `Calc`.
+ - `Accept` the new `C1`, `Gun` column and `Elev` are updated.
+ - *The Drag Coefficient Table will be automatically updated.*
+ - Optionally, `Save Gun` and `Done` in the `GunList`.
+ 
+ img src="{{ site.baseurl }}/img/wiki/feature/atragmx2.png" width="1127" height="600" alt="Calculation" /
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Missing `<` at the beginning and `>` at the end.

File: docs/wiki/feature/atragmx.md [hunk 25]
```diff
@@ -12,21 +12,68 @@ version:
...
+ 
+ - If a new `Target Range` is applied in the `Target` column, select `Drag Coef Table` in the `Options` menu and `Done`.
+ - The ballistic coefficient `C1` and the elevation `Elev` will be recalculated.
+ 
+img src="{{ site.baseurl }}/img/wiki/feature/atragmx3.png" width="1123" height="600" alt="Interpolation" /
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Missing `<` at the beginning and `>` at the end.

File: docs/wiki/feature/atragmx.md [hunk 26]
```diff
@@ -12,21 +12,68 @@ version:
...
+ 
+ ### 3. Official Manual and Horus Videos
+
+[Official Manual](https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf)
```
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Maybe just list those links with a list `- <link>` to condense it a bit.

Comment (J3aLeiNe) [inline docs/wiki/feature/atragmx.md]: Sorry jonpas but i don't understand exactly what you want. 
Dillinger and a french tuto give me this way for a link.
You prefer something like that ? Without `[Official Manual]`?
`- <https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf>`

Or something like that maybe ?
`- [Official Manual]: <https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf>`

(My apologies for my mistakes)

**Comment by GanX44** [comment]
I should really have checked if someone already was working on this. What are the odds that we are missing documentation for the ATragMX for so long and now there are two people working on it.

@J3aLeiNe take a look at https://github.com/acemod/ACE3/pull/4995

I just finished today and submitted the Pull, it should cover everything the ATragMX has to offer including images. Not sure if your Pull or mine will be used for the wiki (or a combination) but i thought that i let you know so you don't spend to much time on it if you don't want to.

Anyways, good to see someone else working on it!

File: docs/wiki/feature/atragmx.md (hunk 26 above)
Comment (jonpas) [inline docs/wiki/feature/atragmx.md]: Just place them in a list, like so:
```
- [Official Manual](https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf)
- [Horus video part1](https://www.youtube.com/watch?v=pg6oqT5jVds)
- [Horus video part2](https://www.youtube.com/watch?v=7SkRnbwoPmw)
```

**Comment by jonpas** [comment]
It would be great if you guys could collaborate on the doc! :)
Slack is open for discussing and working together.

**Comment by J3aLeiNe** [comment]
Missing documentation ? 
Official manual since the AtragMx stand alone (2014):
https://forums.bistudio.com/forums/topic/171882-atragmx-handheld-ballistics-calculator/
Official manual from ACE since 2015:
https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf
There are already all informations in this manual.

**Comment by GanX44** [comment]
@J3aLeiNe  I meant on the ACE Wiki.

@jonpas I would love to, however my Pull includes everything already. I don't want to sound rude but perhaps it would be best to supplement my Pull with J3aLeiNe if needed. Mine is already 360 lines long.

**Comment by J3aLeiNe** [comment]
@jonpas , @bux ,i think i will add something like that: 
## 2. Requirements or Introduction or Preamble ?
### 2.1  Advanced Ballistics: how to enable?
(picture)
### 2.2  Advanced Ballistics: enabled or not in-game?
(picture)
### 2.3 AtragMx: where in the inventory?
(picture)
### 2.4 AtragMx: how to open?
(picture)
### 2.5 Scope module and default zero distance
 - default settings
(picture)
 - example with zero distance 300 meters
(picture)
### 3. Usage
 ... etc ...

I will try to be very basic for all users.
Another idea ?
Question about pictures, there is a limitation of size with the wiki ?

**Comment by jonpas** [comment]
I will merge this PR and you can add to it in another one.

I'd put the Advanced Ballistics section on the Advanced Ballistics wiki page, maybe just link it from ATragMX.

Pictures shouldn't be a problem, just don't overdo them.

**Pull request merged**